benchmark/spmv/spmv_common.hpp

```diff
@@ -66,6 +66,44 @@ void apply_spmv(const char* format_name, std::shared_ptr<gko::Executor> exec,
         auto system_matrix = generator.generate_matrix_with_format(
             exec, format_name, data, &spmv_case[format_name], &allocator);
 
+        // NOTE: I guess should be here?
+        // /* */
```

Comment:
If the following is not required, you can delete them

Reply:
Done!!

Reply:
Done!!